### PR TITLE
add Repeat Interval

### PR DESCRIPTION
**Pull request by toanz**

### CODE REVIEW

**Comment by jeveloper** [comment]
@d4vidi This looks like a great addition, having repeat interval for local notifications.
Please merge :)

**Comment by benevbright** [comment]
any progress?

**Comment by d4vidi** [comment]
@lidanh please have a quick look. @toanz, thanks - I'd really appreciate you also updating the docs!
Last but not least: what about Android?

**Comment by JSafaiyeh** [comment]
Following up on this. What else needs to be done for this feature?

**Comment by christianmellor** [comment]
It would be useful to have this feature, is there anything else we can do or can this be merged now?

**Comment by johnculviner** [comment]
Please merge if good!?

**Comment by manujbahl** [comment]
What is the status on this .. is it available?

**Comment by graphee-gabriel** [comment]
Any news on when this will be merged ?

File: RNNotifications/RNNotifications.m [hunk 1]
```diff
@@ -570,10 +589,17 @@ - (void)handleNotificationActionTriggered:(NSNotification *)notification
...
     if (openedLocalNotification) {
+        UIApplicationState state = [UIApplication sharedApplication].applicationState;
+    
+        NSMutableDictionary* newUserInfo = openedLocalNotification.userInfo.mutableCopy;
+        [newUserInfo removeObjectForKey:@"__id"];
+        openedLocalNotification.userInfo = newUserInfo;
+
         [RNNotificationsBridgeQueue sharedInstance].openedLocalNotification = nil;
-        [RNNotifications didNotificationOpen:openedLocalNotification];
+        // [RNNotifications didNotificationOpen:openedLocalNotification];
```
Comment (sughodke) [inline RNNotifications/RNNotifications.m]: squash

**Pull request closed without merging**